Testscripts/Windows/Run-LksSuite.ps1

```diff
@@ -36,21 +36,20 @@ function Get-TestResult ($currentTestResult, $currentTestData) {
                 $metaData = $_.split(":")[1].replace(' ','').split('[')[0]
                 $result = $_.split("[]")[1]
                 $item = "{0,-35} {1,10}" -f $metaData, $result
-            } elseif ($_ -match "(ok|not ok)+\s+\d+\s+[\w]") {
-                $metaData = $_.split(" ")[2]
-                $result = $_.split(" ")[0]
-
-                if ($result -eq "ok") {
+            } elseif ($_ -match "(ok|not ok)+\s+\d+\s+(selftests:)\s+[\w|\w-\w:]+\s+[\w|\w-\w|.]") {
+                if ($_.Contains("not ok")) {
```

Comment:
why not  ok == skip?

Reply:
The results of 5.4 kernel selftests have this format report "not ok 1 selftests: binderfs: binderfs_test # SKIP". So we check if the line contains "SKIP", if so we flag this test case result as "SKIP".